frontend/src/features/public-form/components/FormFields/FormFieldsContainer.tsx

```diff
@@ -45,11 +44,9 @@ export const FormFieldsContainer = (): JSX.Element | null => {
 
   return (
     <Flex justify="center">
```

Comment:
i wonder if we need an additional wrapper now that we dont have multiple children here. Can we get away with just the Box component (without the Flex)?

Reply:
Youre right, its not needed!